data/jakarta/utils.sh

```diff
@@ -5,125 +5,67 @@ if [ "$(id -u)" != "0" ]; then
-            echo "service $svc has status \"$svcStatus\" but should be active"
-            if [ "$FATAL" = "1" ]; then
-                exit 1
-            fi
-        fi
-    done
-
-    # disabled services
-    for svc in app-service-configurable device-virtual kuiper support-notifications support-scheduler sys-mgmt-agent; do
-        svcStatus="$(snap services edgexfoundry.$svc | grep $svc | awk '{print $2}')"
-        if [ "disabled" != "$svcStatus" ]; then
-            echo "service $svc has status \"$svcStatus\" but should be disabled"
-            if [ "$FATAL" = "1" ]; then
-                exit 1
-            fi
-        fi
-    done
-
-    # inactive services
-    # all the disabled services + the oneshot daemons
-
-    for svc in app-service-configurable device-virtual kuiper security-bootstrapper-redis \
-	security-consul-bootstrapper security-proxy-setup security-secretstore-setup \
-        support-notifications support-scheduler sys-mgmt-agent; do
-        svcStatus="$(snap services edgexfoundry.$svc | grep $svc | awk '{print $3}')"
-        if [ "inactive" != "$svcStatus" ]; then
-            echo "service $svc has status \"$svcStatus\" but should be inactive"
-            if [ "$FATAL" = "1" ]; then
-                exit 1
-            fi
-        fi
-    done
-}
-
-get_snap_svc_status()
-{
-    case "$1" in
-        "status")
-            svcStatus="$(snap services edgexfoundry.$1 | grep $1 | awk '{print $3}')"
-            ;;
-        "")
-            ;;
-    esac
-}
+    # group services by status
 
-snap_remove()
-{
-    snap remove edgexfoundry || true
+    check_enabled_services \
+        `#core services` \
+        "redis \
+        core-data \
+        core-command \
+        core-metadata \
+        `#security services` \
+        kong-daemon postgres vault consul 
+        `#one-shot security services` \
+        security-proxy-setup \
+        security-secretstore-setup \
+        security-bootstrapper-redis \
+        security-consul-bootstrapper "
+
+    check_active_services \
+        `#core services` \
+        "redis \
+        core-data \
+        core-command \
+        core-metadata \
+        `#security services` \
+        kong-daemon postgres vault consul"
+
+    check_disabled_services \
+        `#app service, kuiper and device-virtual`\
+        "app-service-configurable kuiper device-virtual \
+        `#support services, system service` \
+        support-notifications \
+        support-scheduler \
+        sys-mgmt-agent"
+
+    check_inactive_services \
+        `#app service, kuiper and device-virtual `\
+        "app-service-configurable kuiper device-virtual \
+        `#one-shot security services` \
+        security-proxy-setup \
+        security-secretstore-setup \
+        security-bootstrapper-redis \
+        security-consul-bootstrapper \
+        `#support services, system service` \
+        support-notifications \
+        support-scheduler \
+        sys-mgmt-agent"    
```

Comment:
Writing these strings are really error prone. Can you try defining them as arrays?